src/components/SideBar/ItemDrawer/ItemDrawer.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react'
+import { Header, Grid } from 'decentraland-ui'
+import { t } from 'decentraland-dapps/dist/modules/translation/utils'
+import Drawer from 'components/Drawer'
+import AssetCard from 'components/AssetCard'
+import { Props, State } from './ItemDrawer.types'
+import './ItemDrawer.css'
+
+export default class ItemDrawer extends React.PureComponent<Props, State> {
+  state = {
+    listMode: false
+  }
+
+  renderGrid() {
+    const { items } = this.props
+    const { listMode } = this.state
+    let el = []
+
+    for (let i = 0; i < items.length; i += 3) {
```

Comment:
Maybe both `3`s should be in a constant with a name?

Reply:
do you think this is the best way?

Reply:
(I mean looping this way)